packages/pricing/src/migrations/Migration20231101232834.ts

```diff
@@ -30,9 +27,19 @@ export class Migration20231101232834 extends Migration {
     )
 
     this.addSql(
-      'alter table "price_list" add column if not exists "title" text not null, add column if not exists "description" text not null, add column if not exists "type" text check ("type" in (\'sale\', \'override\')) not null default \'sale\', add column if not exists "created_at" timestamptz not null default now(), add column if not exists "updated_at" timestamptz not null default now(), add column if not exists "deleted_at" timestamptz null;'
+      'alter table "price_list" add column if not exists "title" text, add column if not exists "name" text, add column if not exists "description" text not null, add column if not exists "type" text check ("type" in (\'sale\', \'override\')) not null default \'sale\', add column if not exists "created_at" timestamptz not null default now(), add column if not exists "updated_at" timestamptz not null default now(), add column if not exists "deleted_at" timestamptz null;'
```

Comment:
question: This is just a column renaming? Is the model correct?

Reply:
We added this to ensure that pricing migration can take place without exploding from core for both isolated and shared cases. In the core, its called name, but in the module, title. 

We're dropping the name column in the same migration.